src/app/core/entity/entity-actions/entity-actions.service.ts

```diff
@@ -54,6 +61,14 @@ export class EntityActionsService {
         label: $localize`:entity context menu:Delete`,
         tooltip: $localize`:entity context menu tooltip:Remove the record completely from the database.`,
       },
+      {
+        action: "edit",
+        execute: (e, nav) => this.edit(e, nav),
+        permission: "edit",
+        icon: "edit",
+        label: $localize`:entity context menu:Edit`,
+        tooltip: $localize`:entity context menu tooltip:Update the record.`,
+      },
```

Comment:
This adds it to the context menu of a single entity details view. I think we don't need to add "bulk edit" feature here at all. (not very clearly documented code however ...)